l10n_eu_oss/readme/ROADMAP.rst

```diff
@@ -1 +1,5 @@
 #. This module doesn't take into account if one or more products have different type of taxes in the destination contry.
+
+#. There are some countries (for example Spain) that have different taxes dependieng on the region (Canary islands). These use cases haven't been considered.
+
+#. There are some countries (for example Cyprus) that haven't very common reduced taxes dependieng on the typolgy of the sold product. In Cyprus, it happens with *Supply of goods and services of a kind normally intended for use in agricultural production but excluding capital goods such as machinery or buildings* where an 8% of VAT has to be applied instead of the common reduced type. These use cases haven't been considered.
```

Comment:
```suggestion
#. There are some countries (for example Cyprus) that haven't got very common reduced taxes depending on the typology of the sold product. In Cyprus, it happens with *Supply of goods and services of a kind normally intended for use in agricultural production but excluding capital goods such as machinery or buildings* where an 8% of VAT has to be applied instead of the common reduced type. These use cases haven't been considered.
```

Reply:
Incorporated your fixes and improved the wording a bit further